logstreams/src/main/java/io/zeebe/logstreams/impl/LogStorageAppender.java

```diff
@@ -139,25 +138,30 @@ private void appendToPrimitive(long appendIndex, byte[] bytesToAppend, long last
               actor.run(
                   () -> {
                     if (error != null) {
-                      LOG.trace(
+                      LOG.debug(
```

Comment:
Do you think it makes sense to put them on debug level?

Reply:
No. Sorry. It's left over from debugging.